### PR TITLE
Jra config psd01

### PR DESCRIPTION
**Pull request by jamsden**
Updates requested by OASIS.

### CODE REVIEW

File: specs/config/README.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ Configuration Management Technical Committee. Those previous drafts may be found
...
+-   oslc-config-mgt.html
+-   versoned-resources.html
 
 The following files are additional material of interest, but not part of the specification.
 Some files may become Project Notes or similar documents.
 
-- README.md (this file)
-- motivation.html
+-   README.md (this file)
+-   motivation.html
```
Comment (berezovskyi) [inline specs/config/README.md:26]: can be removed, I deleted the file

File: specs/config/config-resources.html [hunk 2]
```diff
@@ -6,15 +6,15 @@
       name="description"
       content="This part of the OSLC Configuration Management Specification defines the representation and behavior of components and configurations of resources in those components (typically referred to as configurations of components, or just configurations)."
     />
-    <title>OSLC Configuration Management 1.0 Part 3: Configuration Specification</title>
+    <title>OSLC Configuration Management Version 1.0. Part 3: Configuration Specification</title>
     <script
-      src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.22/builds/respec-oasis-common.min.js"
+      src="https://cdn.jsdelivr.net/gh/oasis-tcs/tab-respec@v2.1.28/builds/respec-oasis-common.min.js"
```
Comment (berezovskyi) [inline specs/config/config-resources.html:11]: 29 please, it has the new logo.

File: specs/config/config-resources.html [hunk 3]
```diff
@@ -261,7 +304,22 @@ <h4>Configuration context</h4>
           >
         </li>
       </ul>
-      <p>TBD - define syntax of Configuration-Context header by referring to IETF RFC.</p>
+      <div class="conformance">
```
Comment (berezovskyi) [inline specs/config/config-resources.html:307]: I thought you are making just a few more edits on top of the existing PR? Why are these not in the master branch yet?

**Review by berezovskyi — CHANGES_REQUESTED** [review]
Looks good but can you please update the ReSpec version and, more importantly, why are material changes showing up in the diff? They look right and I thought they were on master already. For example, ABNF is there: https://oslc-op.github.io/oslc-specs/specs/config/config-resources.html##cc-the-configuration-context-header-is-a-uri-that-must-conform-to-the-following-abnf-grammar-abnf-sp-htab-x09-wsp-sp-htab-header-config-context-config-context-configuration-context-wsp-uri-where-the-term-uri-is-defined-in-uri-.x Please check that your diff is correct.

**Comment by jamsden** [comment]
I reused an old branch - shouldn't have done that.

**Review by berezovskyi — APPROVED** [review]

**Pull request merged**